edx_django_utils/management/docs/decisions/0001-user-and-group-management-commands.rst

```diff
@@ -0,0 +1,20 @@
+User and Group Management Commands
+===============================
+
+Context
+-------
+
+Open edX administrators ocassionally need to grant permissions to users for certain django services (e.g. ecommerce, registrar). Right now, this is done manually via django admin of the django app. This method of granting permissions is not endorsed.
+
+The user and group management commands in edx-platform, are supposedly generic enough to be used via a common library/app so that users and groups can be managed without much (repeated) modificaiton to IDAs.
```

Comment:
```suggestion
The user and group management commands in edx-platform, are supposedly generic enough to be used via a common library/app so that users and groups can be managed without much (repeated) modification to IDAs.
```